core/templates/dev/head/player/exploration_player.html

```diff
@@ -95,7 +96,8 @@ <h2 class="oppia-info-card-title"><[explorationCardTitle]></h2>
               src="/third_party/static/material-design-icons-1.0.1/ic_group_black_48dp.png">
             <li ng-repeat="ContributorUsername in explorationContributorUsernames| limitTo: 2"
               tooltip="<[ContributorUsername]>" tooltip-placement="top">
-              <img class="oppia-contributors-circle" ng-src="/images/avatar/user_mint_48px.png" alt="">
+              <profile-link-image username="<[ContributorUsername]>" classes="oppia-contributors-circle" alt="">
```

Comment:
I think it's fine not to pass 'alt' -- let's keep it simple for now. The directive can just automatically have `alt = ""`.

Also, can we get away without passing 'classes'? What happens if you don't? I think it might be better for the "make the image round" class to be declared within the directive, so that the directive ends up being fully self-contained. You don't necessarily need to have full visual parity with what existed before, as long as the end result looks nice.